.github/workflows/testsuite.yml

```diff
@@ -31,4 +29,16 @@ jobs:
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
-      - run: cd tests && meteor npm install && meteor npm run test
+
+      - name: install dependencies
+        run: |
+          sudo apt update -y
+          apt search libnss3
+          sudo apt install -y libnss3 libatk-bridge2.0-0        
```

Comment:
Can you please explain a bit more what these dependencies are about?

Reply:
To launch workflows locally I need to install the chromium dependency, I remove it.